src/main/java/org/folio/rest/service/report/context/HasAccountInfo.java

```diff
@@ -9,7 +9,10 @@
 
 public interface HasAccountInfo {
   Account getAccountById(String accountId);
+
   Future<Void> updateAccountContextWithActions(String accountId, List<Feefineaction> actions);
+
   boolean isAccountContextCreated(String accountId);
+
```

Comment:
Do we always put new lines between fields in classes?

Reply:
We do have lines between methods